presentation/src/main/java/com/architect/coders/mu8/Extensions.kt

```diff
@@ -0,0 +1,8 @@
+package com.architect.coders.mu8
+
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.loadUrl(url: String) {
```

Comment:
@SalosV Do you want to use a placeHolder for the image or how can we manage if glide has an error while the image is loading from URL? (callback) ? 
@recaldev